web-ui/src/components/team-results/TeamResults.css

```diff
@@ -1,3 +1,29 @@
 .MuiCard-root {
-    margin-bottom: 1rem;
+  margin-bottom: 1rem;
+}
+
+.summary-card {
+  max-height: 15rem;
+  max-width: 45%;
+  min-width: 45%;
```

Comment:
pfffft

Reply:
Hey you wanted them to look like that lol